internal/pkg/workspace/workspace.go

```diff
@@ -28,9 +28,10 @@ import (
 const (
 	// CopilotDirName is the name of the directory where generated infrastructure code for an application will be stored.
 	CopilotDirName = "copilot"
+	// WorkspaceSummaryFileName is the name of the file that is associated with the application
```

Comment:
nit: end the comments with a period pls https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Please deduct -100 points for every I make this mistake -💯